### PR TITLE
refactor(insights): remove filter based card content

### PR DESCRIPTION
**Pull request by thmsobrmlr**
## Problem

We have feature flag `query-based-dashboard-cards` that replaces filter based cards with their query based equivalent in most cases (i.e. in contexts where the feature flag works). We'd like to remove the legacy code and roll out query based cards for all occurances.

## Changes

- Removes the legacy `FilterBasedCard` component.
- Fixes a bug where an insight with many breakdown values would show the "Load more breakdown values..." in the card view.

## How did you test this code?

CI run and clicking around

### CODE REVIEW

File: frontend/__snapshots__/exporter-exporter--lifecycle-insight--dark.png
Comment (thmsobrmlr) [inline frontend/__snapshots__/exporter-exporter--lifecycle-insight--dark.png:1]: I did manually export one insight of each kind and compared it against the previous version - couldn't replicate the shrinking as seen in these snapshot tests e.g. this is the lifecycle insight with the new query based card:
export-lifecycle

File: cypress/e2e/insights-date-picker.cy.ts [hunk 1]
```diff
@@ -1,8 +1,11 @@
+import 'cypress-network-idle'
+
 import { urls } from 'scenes/urls'
 
 describe('insights date picker', () => {
     beforeEach(() => {
         cy.visit(urls.insightNew())
+        cy.waitForNetworkIdle(2000)
```
Comment (thmsobrmlr) [inline cypress/e2e/insights-date-picker.cy.ts:8]: TODO: Try this with 300ms.

File: cypress/e2e/insights-date-picker.cy.ts [hunk 2]
```diff
@@ -1,8 +1,11 @@
+import 'cypress-network-idle'
+
 import { urls } from 'scenes/urls'
 
 describe('insights date picker', () => {
     beforeEach(() => {
         cy.visit(urls.insightNew())
+        cy.waitForNetworkIdle(300)
```
Comment (thmsobrmlr) [inline cypress/e2e/insights-date-picker.cy.ts:8]: This was much easier than mocking requests to `/query/` and `/query/:queryId`, which I couldn't get working.

**Review by skoob13 — APPROVED** [review]
The PR looks good! I didn't notice any problems with cards, exports, or embeddings.

**Pull request merged**